content/blog/2021-03-15-March-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,195 @@
+﻿---
+title: March '21 Heartbeat
+date: 2021-03-15
+description: |
+  Monthly updates are here! News, use cases, videos, journal publications and more!
```

Comment:
please keep it wrapped at 80 symbols, we don't any automation now for this :(. In this case it should something like

```
  Monthly updates are here! News, use cases, videos, journal publications and
  more!
```

probably